app/views/powerbanks/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Available powerbanks</h1>
+
+<ul>
+  <% @powerbanks.each do |powerbank| %>
+    <li><%= powerbank.description %></li>
+    <li><%= powerbank.price %></li>
+    <li><%= powerbank.accessories %></li>
```

Comment:
We will need to add in a name separate to the description as we will need to pull the name for the dashboard and index pages
There will also need to be an indication of availability when they are listed

Reply:
Thanks @demisaurusRex 
I can do that on thursday and then we can review again